BrowserKit/Tests/TabDataStoreTests/TempTests.swift

```diff
@@ -0,0 +1,12 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import XCTest
+@testable import TabDataStore
+
+final class TempTests: XCTestCase {
```

Comment:
Was a test for what?

Reply:
Sorry, meant to leave a comment for this. It's a temp test file because BrowserKit was complaining about the target having no source files in the directory. Will replace with real tests once I add any code with logic and not just models.